opus/src/components/formPostOffer/FormPostOffer.js

```diff
@@ -1,9 +1,14 @@
 import React from 'react';
 import axios from 'axios';
 import Footer from '../footer/Footer';
+import { connect } from 'react-redux';
 
 import './FormPostOffer.css';
 
+const mapDispatchToProps = state => ({
```

Comment:
This provides a value from the state to the component, no dispatch - better name this mapStateToProps